oe_whitelabel.theme

```diff
@@ -155,6 +155,23 @@ function oe_whitelabel_preprocess_page(&$variables) {
   $variables['site_logo_href'] = $site_logo_href;
 }
 
+/**
+ * Implements hook_preprocess_html().
+ */
+function oe_whitelabel_preprocess_html(array &$variables) {
+  $cck_container = [
+    '#type' => 'html_tag',
+    '#tag' => 'div',
+    '#attributes' => [
+      'id' => 'cck_here',
+      'role' => 'alertdialog',
+      'aria-live' => 'assertive',
+    ],
+  ];
+
+  $variables['page_top']['cck_container'] = $cck_container;
```

Comment:
make sure its the first one

Reply:
it's not possible because the drupal tests will fail on the admin toolbar not finding specific elements.